macaroon.js

```diff
@@ -7,14 +7,27 @@ nacl.util = require('tweetnacl-util');
 const NONCELEN = 24;
 
 /**
-  Check if supplied value is a string. Throws if not.
+  Check if supplied value is a string and return it. Throws if not.
   @param {String} val The value to assert as a string
   @param {String} msg The value label.
```

Comment:
Plz add 

`@returns {String} The supplied value.`

Reply:
done